source/common/common/macros.h

```diff
@@ -54,4 +54,7 @@ namespace Envoy {
 #define FALLTHRU
 #endif
 
+#if (defined(__GNUC__) && !defined(__clang__))
```

Comment:
This should be in include/envoy/common/platform.h?

Reply:
Hmm, the comment in platform.h says:
```
"platform.h" header exists to simplify the most common references
to non-ANSI C/C++ headers, required on Windows, Posix, Linux, BSD etc,
and to provide substitute definitions when absolutely required.
``` 
feels like not a good spot to me for the macro in question?

Reply:
OK, I don't feel strongly either.